pipeline/src/node_properties/track_properties.rs

```diff
@@ -20,3 +21,13 @@ pub enum Tracker {
     /// bounding boxes across frames to determine the object’s unique ID.
     Iou,
 }
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(tag = "type", rename_all = "snake_case")]
+pub enum TrackerProfile {
```

Comment:
If you implement [Default](https://doc.rust-lang.org/std/default/trait.Default.html) for this, returning `Self::Default`, you can use `serde(default)` to deserialize a missing property to that.